NBitcoin.TestFramework/WellknownNodeDownloadData.cs

```diff
@@ -844,6 +846,41 @@ public class ChaincoinNodeDownloadData
 			};
 		}
 
+		/// <summary>
+		/// Using Stratis C# full node.
+		/// Should be updated to use official release once it is deployed.
+		/// </summary>
+		public class StratisNodeDownloadData
+		{
+			public NodeDownloadData v3_0_0 = new NodeDownloadData()
+			{
+				Version = "3.0",
+				Windows = new NodeOSDownloadData()
+				{
+					DownloadLink = "https://github.com/mikedennis/StratisBitcoinFullNode/releases/download/{0}/stratis-{0}-win64.zip",
+					Archive = "stratis-{0}-win64.zip",
+					Executable = "stratis-{0}-win64/Stratis.StratisD.exe",
+					Hash = "7B0ABEA75B032D8FAF3BEE071A892E2864A31A8ECC42F7AA4300CC51B1CA1D5A"
+				},
+				Linux = new NodeOSDownloadData()
+				{
+					DownloadLink = "https://github.com/mikedennis/StratisBitcoinFullNode/releases/download/{0}/stratis-{0}-linux64.tgz",
```

Comment:
not officially released?

Reply:
All my Stratis changes have been pushed to master now but binaries have not yet been released.

Reply:
We're going to build some releases coming from the Stratis repo.